kafka/monitoring-kafka-performance-metrics.md

```diff
@@ -9,7 +9,8 @@ There are a few key differences between Kafka and other queueing systems like [R
 3. It is [very fast][kafka-benchmark], even in a small cluster. 
 4. It has strong ordering semantics and durability guarantees.
 
-Despite being pre-1.0, (current version is [0.9.0.1][download-kafka]), it is [production-ready][user-list], and powers a large number of high-profile companies including [LinkedIn][linkedin-msgs], Yahoo, Netflix, and Datadog.
+Kafka is [production-ready][user-list], and [powers a large number of high-profile companies][user-list] including [LinkedIn][linkedin-msgs], Yahoo, Netflix, and Datadog.  
```

Comment:
I don't think we need to specifically state that it's prod-ready since it's no longer pre-1.0. Would suggest: 
>Kafka [powers a large number of high-profile companies][user-list]

Reply:
Ack